code/modules/urist/items/saxophone.dm

```diff
@@ -0,0 +1,18 @@
+/obj/item/device/saxophone
+	name = "saxophone"
+	desc = "A shiny brass saxophone."
+	icon = 'icons/urist/items/tools.dmi'
+	icon_state = "saxophone"
+	item_state = "saxophone"
+	urist_only = 1
+	force = 10
```

Comment:
That seems way too powerful for basically a joke item. IIRC *fire extinguishers and full airtanks* deal less.